corehq/apps/registration/utils.py

```diff
@@ -105,7 +105,7 @@ def activate_new_user(
     return new_user
 
 
-def request_new_domain(request, project_name, is_new_user=True):
+def request_new_domain(request, project_name, is_new_user=True, is_new_sso_user=False):
```

Comment:
Is it possible to de-couple registration from the notion of SSO? It seems the new feature that SSO requires from registration is that user emails do not need verification. Could we change `is_new_sso_user` to `skip_email_validation`, and then send a more generic email? This keeps `request_new_domain independent of SSO, and lets us re-use this code if some other process needs to skip verification.